src/index.html

```diff
@@ -4,10 +4,305 @@
+            <a
+              href="https://goo.gl/maps/SFF4Et6QTFyH1CoE9"
+              class="firstPage__link">
+              here
+            </a>
+          </h2>
+        </div>
+
+        <div class="firstPage__info">
+          <h3>Over 50 years experience in love.</h3>
+
+          <p>Mazel and VA Tsukkerman in his "Analysis of musical works." Pointillism,
+            which originated in the music of the early twentieth century, microforms,
+            found a distant historical.
+          </p>
+
+          <p>The market leader in real estate since 2003. Awards Real Estate Company,
+            2010, 2011, 2013. More than 200 satisfied customers premium segment. We focus
+            only on new buildings and high-level doskanalno know this area better than
+            other specialists.
+            <br>
+            Working directly with developers - so have more useful
+            information about the profitable deals.
+          </p>
+
+          <p class="firstPage__infoBold">We will never be late!
+            <br>
+            Guaranteed secure transaction - 100%
+          </p>
+        </div>
+      </div>
+
+      <div class="secondPage" id="gallery">
+        <div class="secondPage__header">
+          <h2 class="secondPage__title">Gallery</h2>
+          <a href="#home" class="secondPage__logo"></a>
+        </div>
+
+        <div class="secondPage__box">
+          <div class="secondPage__image1 image"></div>
+          <div class="secondPage__image2 image"></div>
+        </div>
+        <div>
+          <a
+            href="https://goo.gl/maps/M6K1VtBfGHo27UKX8"
+            class="secondPage__link"
+          >
+          Marina Palms
+          </a>
+          / North Miami Beach, FL 33162
+        </div>
+
+        <div class="secondPage__box">
+          <div class="secondPage__image3 image"></div>
+          <div class="secondPage__image4 image"></div>
+        </div>
+        <div>
+          <a
+            href="https://goo.gl/maps/v5VU1a9aiWzWSTnk8"
+            class="secondPage__link"
+          >
+          Edition Residens
+          </a>
+          / Miami Beach, FL 33139
+        </div>
+
+        <div class="secondPage__box">
+          <div class="secondPage__image5 image"></div>
+          <div class="secondPage__image6 image"></div>
+        </div>
+        <div>
+          <a
+            href="https://goo.gl/maps/UNu1oRGBBiYjoVbz9"
+            class="secondPage__link"
+          >
+            Faena House
+          </a>
+             / Miami Beach, FL 33140
+        </div>
+
+        <div class="secondPage__box">
+          <a href="#" class="secondPage__button">View all</a>
+        </div>
+      </div>
+
+      <div class="thirdPage" id="testimonials">
+        <div class="thirdPage__header">
+          <h2 class="thirdPage__title">Testimonials</h2>
+          <a href="#home" class="thirdPage__logo"></a>
+        </div>
+
+        <div class="thirdPage__peopleBox">
```

Comment:
```suggestion
        <div class="third-page__people-box">
```
Let's use kebab-case for class names